### ISSUE TITLE
Restructuring program

### ISSUE THREAD

**Issue by akilan98** [CircuitRunners/grits_2017#3]
Definition of done:
- [x] Project files/folders match the structure found [here](https://github.com/CircuitRunners/getting_started/blob/master/project_structure.md)

**Comment by RazorXFire** [comment]
Id love to fix the structure come FRC time, but I have the command/subsystem working and the competition is this weekend.

**Comment by akilann** [comment]
Fair enough but the PR should be a good example of what I was going for.

**Comment by akilan98** [comment]
Fair enough but the PR should be a good example of what I was going for. Also upload what you have so I can see it.

**Comment by RazorXFire** [comment]
I did, everything should be there in the master branch
